### PR TITLE
Better visual and UI

### PR DESCRIPTION
**Pull request by jasonlemay**

### CODE REVIEW

File: client/lib/components/create_game_component.html [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+        <div class="group-actions">
+            <label><span>Game name</span> <input type="text" name="gameName" ng-model="gameName"></label>
+            <label><span>Password</span> <input type="text" name="password" ng-model="password"></label>
+            <div class="group-actions-btn">
+                <input type="submit" value="Create"/>
+            </div>
+        </div>
+        
+    </div>
+</div>
```
Comment (Pacane) [inline client/lib/components/create_game_component.html]: nl eof

File: client/web/pokerplanning.css [hunk 2]
```diff
@@ -30,18 +30,50 @@ body {
...
+    }
+    
+    #loggedIn {
+        position: relative;
+        top: 0;
+        left: 0;
+        right: auto;
+        bottom: auto;
+    }
+}
```
Comment (Pacane) [inline client/web/pokerplanning.css]: nl eof

File: client/lib/components/login_component.dart [hunk 3]
```diff
@@ -37,6 +43,19 @@ class LoginComponent implements ScopeAware, ShadowRootAware, AttachAware {
 
   void onShadowRoot(ShadowRoot shadowRoot) {
     this.shadowRoot = shadowRoot;
+    checkDisplayState();
+  }
+
+  void checkDisplayState() {
+    if(_session.userName != null) {
```
Comment (Pacane) [inline client/lib/components/login_component.dart]: remove the if for this --> `.classes.toggle(className, _session.userName != null)`

File: client/lib/components/core.css [hunk 4]
```diff
@@ -21,28 +19,141 @@ h1 span {
...
+    font-size: 1.2rem;
+    
+    display: block;
+    width: 100%;
+    padding: 1rem;
+    border: 1px solid #5b5e6d;
+}
+
+.group-actions input:focus {
+    outline: 0;
```
Comment (LarryMatte) [inline client/lib/components/core.css]: move outline after bg-color and add space

File: client/lib/components/core.css [hunk 5]
```diff
@@ -1,14 +1,12 @@
...
-    box-sizing: border-box;
-}
-
-h1 {
+h1,
+h1 a {
     font-size: 3rem;
     text-transform: uppercase;
+    color: #fff;
+
```
Comment (LarryMatte) [inline client/lib/components/core.css]: rmv space

File: client/lib/components/my_card.css [hunk 6]
```diff
@@ -32,13 +32,13 @@
 
 .card:hover {
     padding-bottom: 10px;
```
Comment (LarryMatte) [inline client/lib/components/my_card.css]: move padding after color and add space

File: client/web/pokerplanning.css [hunk 7]
```diff
@@ -30,18 +30,50 @@ body {
...
-    bottom: -0.3em;
-    right: 0;
+    bottom: 1em;
+    right: 11px;
+}
+
+#loggedIn a {
+    color: #eee;
+    font-size: 80%;
+
```
Comment (LarryMatte) [inline client/web/pokerplanning.css]: rmv space

**Pull request merged**